localrec/protocols/__init__.py

```diff
@@ -24,6 +24,6 @@
 # *
 # **************************************************************************
 
-from protocol_localized import ProtLocalizedRecons
-from protocol_localized_extraction import ProtLocalizedExtraction
-from protocol_filter_subparticles import ProtFilterSubParts
+from localrec.protocols.protocol_localized import ProtLocalizedRecons
```

Comment:
Here the import should be local:
from .protocol_xxx import ProtXXX

Here you are inside teh protocols submodule of the package/plugin...

Reply:
Thank JoseMi, I'll change thats